src/components/toggle/toggle.html

```diff
@@ -0,0 +1,13 @@
+<label [attr.for]="inputId" class="md-toggle-label">
```

Comment:
I'm thinking we should probably rename from just `toggle` to `toggle-button`, which is more explicit. Anyone else have thoughts on this?